src/contract.rs

```diff
@@ -32,6 +32,7 @@ pub fn instantiate(
         recovery_admin: msg.recovery_admin,
         lp: msg.lp.clone(),
         capital_denom: msg.capital_denom,
+        capital_per_share: msg.capital_per_share,
```

Comment:
Do we truly need to copy the price per shares onto these subscription contracts? It seems a little weird to have it duplicated from the parent raise contract in that it technically "could" be different per subscription contract (the way it is currently implemented) if the raise contract allowed it. Could we just pass the price per share on the messages coming from the capital raise contract instead of needing to store it in the subscription's internal state?

Reply:
this or we have sub contract querying the raise for it...

the logic enforcing evenly divisible is not duplicated, raise enforces a gp specified min / max commitment, target, and accepted; while the sub enforces lp specified min / max commitment and issued capital call amount

Reply:
it comes down to dupe data vs extra queries during contract execution